src/store/messaging.store.ts

```diff
@@ -162,10 +169,16 @@ export const useMessagingStore = create<MessagingState>((set, g) => {
 
       const knownEventIds = new Set(oooc.events.map((e) => e.id.split("_")[1]));
       const newIndexerPayments = indexerPayments.filter(
-        (p) => !knownEventIds.has(p.tx_id)
+        (p, idx, self) =>
+          !knownEventIds.has(p.tx_id) &&
+          // @todo: remove when indexer dedup filter
```

Comment:
todo: wait for indexer changes

Reply:
We can remove these check and consider that https://github.com/K-Kluster/kasia-indexer/pull/16/ will be merged soon